include/SFML/Window/Window.hpp

```diff
@@ -197,6 +197,14 @@ class SFML_WINDOW_API Window : public WindowBase, GlResource
     ////////////////////////////////////////////////////////////
     const ContextSettings& getSettings() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the time between the last two display() calls
```

Comment:
Nitpicking, but it is not clear if calling this is allowed before display has been called tow times, and what it does if it is allowed.

Reply:
That's a good point. This edge behavior is not necessarily intuitive.